f0: 125
BW: 0
Ns: 200
I_true: true
Q_true: true
Ntar: 4
MTI: 0
Mth: 0
Rmax: 100
Alpha: 5
distance_true: false
velocity_true: true
SNR_true: true
movement_true: false
mode: 1